e2e-tests/kuttl-test.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
```

Comment:
No CRD installed as a pre-step?

Reply:
Having the `crdDir` field had no effect for me, I'm installing the CRD manually before deploying the operator